.github/workflows/codeql-analysis.yml

```
@@ -11,6 +11,9 @@
 #
 name: "CodeQL"
 
+permissions:
+  security-events: write
```

Let's be completely explicit
```suggestion
permissions:
  actions: read
  contents: read
  security-events: write
```

---

Sure, thanks